src/lib/components/product.svelte

```diff
@@ -40,8 +40,11 @@
     margin-bottom: 20px;
   }
   @media (hover: hover) {
-    .product:hover {
+    /* .product:hover {
```

Comment:
when did we decide to remove this?

Reply:
I removed it because I was making an animation that would show when hovering that would show the technical sheet was a test

Reply:
But you've kept the cursor pointer - delete that as well then, since it's not clickable...